styles.css

```diff
@@ -0,0 +1,57 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  padding-top: 50px;
+}
+
+h2 {
+  font-family: "Playfair Display", serif;
+  font-weight: 400;
+  font-size: 70px;
+  color: #1a2236;
+  padding-bottom: 30px;
+  padding-left: 16px;
+}
+
+.hello {
+  font-family: "Poppins", sans-serif;
+  padding-bottom: 50px;
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+#menu {
+  margin-right: 9px;
+}
+
+#collab button {
+  background-color: #1a2236;
+  color: white;
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+  width: 400px;
+  height: 70px;
```

Comment:
- [ ] Some of the styling measurements used here are not accurate, consider referring to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) for the exact measurements.